devel/zinc/Portfile

```diff
@@ -12,15 +12,16 @@ long_description \
 platforms       darwin
 supported_archs noarch
 
-master_sites    http://downloads.typesafe.com/zinc/${version}
+master_sites    https://downloads.typesafe.com/zinc/${version}
 
 use_configure   no
 build           {}
 
 extract.suffix  .tgz
-checksums       md5    a1e71c996bfc875acaa32ea192ef8df8 \
-                sha1   ff91434ce32ba4fcfadb8621896edfd271a9bd21 \
-                sha256 92bdeb4d1ba1a08f8a3aa1a0b10d36a06060c47f16fc523856f3ada598362020
+checksums       md5     120fc323649a6f83a755edbd052e9851 \
+                sha1    c60395360b7a527362e8149e68b12d364c4ee9eb \
```

Comment:
You should remove the md5 and sha1 checksums and add rmd160.

Reply:
Some upstream provides only the weaker type of checksum, so I tend to keep them. My current understanding is that `sha256` and `size` are strongly recommended, and at least one other checksum is needed.

Reply:
We always put in rmd160 these days, and `port -v checksum` will always report it to you.

Reply:
(As the other checksums can be faked at this point there's no real point in keeping them, but I won't object overly much.)

Reply:
I like keeping the upstream checksums if they provide them.

Reply:
Yes, drop the md5 and sha1 as the upstream doesn't provide them and add the rmd160. I like providing the upstream checksums to make it easier for a reviewer to see that we haven't simply tried to substitute another tarball.

Reply:
@blair Done.

P.S. @pmetzger When there's no consensus, the port maintainer sets the rules.

Reply:
> It's even been reified in the newest port lint (not yet released I think).

I think you are referring to https://github.com/macports/macports-base/commit/f32bca0e2d5a458ff2e4dcca65d9a155202b2841, which only warns about missing `size`, not other checksums.

Anyway, let's not discuss too much on a closed PR.